content/articles/understanding-the-resilient-packet-ring-technology/index.md

```diff
@@ -0,0 +1,49 @@
+
+### Adopted types of Transmission Media
+
+Media Access Control panels are the hardware required for interaction with wireless or wired transmission medium. It has a logical link control associated with the data link layer that offers control of the flow and multiplexing of the medium of transmission. During data transfers, MAC captures frames adding synchronization for effective transmission. A code for error detection is then added to the frame. MAC also compensates collisions through re originating transmission after detection of a jam signal. It ensures data integrity is maintained through the verification of the sequence for the sender’s frame check.
+
+### Application of Packet Ring Technology
+
+RPR is commonly used in national SONET, metropolitan, and ethernet carrier networks. This architecture is normally incorporated into the SONET networks so that they can gain the ability to transfer packets. This is because, the technology, RPR, offers the network statistical multiplexing that is fault-tolerant. To achieve this level of fault tolerance, a resilient packet ring employs the use of two counter-rotating rings. Both of these rings are connected by fiber forming a ring topology. The nodes forming the RPR on the other hand both transmit packets in opposing directions. The two nodes, that is inner and outer rings, function simultaneously and in the event, one fails only the capacity of traffic transferred is decreased but there is no unavailability. 
+
+### Traffic queues and Class of Service
+
+The traffic present is associated with Class of Service, which is categorized into classes A, B, and C. They signify high, medium, and low traffic simultaneously. Class A is a virtual circuit bandwidth that offers support to low latency applications such as video. Class B is a combination of the bandwidth for the virtual circuit as well as the intensity of the burst that is higher than the CIR. Class C is the most effective effort traffic that is highly flexible as it integrates any available bandwidth supporting the access traffic of the internet.  

```

Comment:
How is this section connected with the topic of your article? Give further explanations to connect it with the topic of the article. If it is a sub-section, then you should use a level four header.